pkg/api/dtos/dashboard.go

```diff
@@ -34,6 +34,7 @@ type DashboardMeta struct {
 	ProvisionedExternalId      string                `json:"provisionedExternalId"`
 	AnnotationsPermissions     *AnnotationPermission `json:"annotationsPermissions"`
 	PublicDashboardAccessToken string                `json:"publicDashboardAccessToken"`
+	HasPublicDashboard         bool                  `json:"hasPublicDashboard"`
```

Comment:
Is this quite the right name? Do we care if they have one but it's disabled? maybe this should be "sharedPublicly" or "publicDashboardEnabled"

Reply:
currently, this represents `do they have at least 1 enabled public dashboard`

Reply:
How about `PublicDashboardEnabled` ?

Reply:
It would be useful to know if a dashboard has a public dashboard to render the sharing tab (we are not using this flag for that now because it has another meaning). I think, if we are populating this at DashboardMeta maybe we can use it in the future instead of going to the DB again when the tab is rendered. I don't think we can represent all of these cases with one boolean (has a public dashboard or not, the public dashboard is enabled or not). Maybe we need an object to start representing all of these possibilities.